### ISSUE TITLE
Goodnesskay\LaravelPdfViewer\LaravelPdfViewerServiceProvider not found

### ISSUE THREAD

**Issue by acv-thanhnv** [goodnesskay/LARAVEL-PDF-VIEWER#9]
I saw other people and me has this error, please resolve it, thanks .

**Comment by siiamine** [comment]
why the owner doesnt reply ????

**Comment by austinatp2** [comment]
1. Remove Goodnesskay\LaravelPdfViewer\LaravelPdfViewerServiceProvider::class, from config/app.php, providers key, 
2. Copy folder laraview 
> from: vendor\goodnesskay\laravelpdfviewer
> to: public
3. Copy pdf file (whatever), ex test.pdf, to public folder
4. Test, open Browser
> http://xxx.xxx.xx/laraview/#../test.pdf

**Comment by bagusindrayana** [comment]
yeah i gues this package just copy laraview folder to public folder

ohh yeah if you use idm or similar maybe you want disable that because idm always download pdf file except open in browser

**Comment by abdulrehman25** [comment]
@austinatp2 Answer worked, thanks Man but it's not showing the file content and auto-downloading the file

Below is the SS attached.

![image](https://user-images.githubusercontent.com/51014352/103294591-f9294e80-4a13-11eb-8256-f98966e20bf7.png)

**Comment by davcpas1234** [comment]
Hey everyone 👋🏻 

I ran into this issue myself and it's to do with the way that the `composer.json` file is constructed. I have made a fork of the project and resolved this issue in my Stable v1.0.1 version. You can view it here: [https://github.com/davcpas1234/laravel-pdf-viewer](https://github.com/davcpas1234/laravel-pdf-viewer)
